tests/monitoring/monitoring.go

```diff
@@ -637,8 +642,110 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 		})
 	})
+
+	Context("System Alerts", func() {
+		AfterEach(func() {
+			waitUntilAlertDoesNotExist("KubeVirtNoAvailableNodesToRunVMs")
+		})
+
+		disableVirtHandler := func() *v1.KubeVirt {
+			originalKv := util.GetCurrentKv(virtClient)
+			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(originalKv.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			kv.Spec.CustomizeComponents = v1.CustomizeComponents{
+				Patches: []v1.CustomizeComponentsPatch{
+					{
+						ResourceName: virtHandler.deploymentName,
+						ResourceType: "DaemonSet",
+						Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
+						Type:         v1.StrategicMergePatchType,
+					},
+				},
+			}
+
+			kv, err = virtClient.KubeVirt(originalKv.Namespace).Update(kv)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() string {
```

Comment:
You probably want to verify there are no virt-handlers hanging around.

Reply:
added